### DISCUSSION TITLE
Possible refund for GitHub copilot?

### DISCUSSION THREAD

**Discussion by yiluansong** [community/community#131955] [GitHub Education]
### Hi everyone,

I paid for one month of GitHub Copilot without knowing that the benefit for teachers include free subscription. I updated my education email and re-verified immediately upon learning about it. My document uploaded can show that my employment covers the period of my paid subscription. Is it possible to get a refund of my one-month subscription? Thank you!

**Comment by paiineli** [comment]
Hi,

Thank you for reaching out. I understand your situation regarding the GitHub Copilot subscription. Since you have updated your educational email and verified your status as a teacher, I recommend contacting GitHub Support directly to inquire about a refund for your one-month subscription.

Please provide them with your proof of employment and any relevant details about your subscription. They should be able to assist you further.

If you need help drafting your message to support, let me know!

Best,
LPG

**Comment by davevad93** [comment]
Hi @yiluansong ,

I don't know if it's possible to get a refund, you could try asking support by opening a ticket [here](https://support.github.com/contact/account), select `Billing, payments or receipts` and describe your issue.

**Comment by samus-aran** [comment]
Hi @yiluansong 👋 

Copilot refund request may be eligible to be handled by our virtual assistant which would get this processed for you immediately.

**Can you please try following these steps?:**

- Follow this specific link to our contact form: https://support.github.com/contact/account?flow=refund_copilot_individual
- Scroll down a bit to see a blue button activated to "Try our virtual agent".
- Follow the prompts to request the cancellation and prorated refund.

This should automatically process the refund and cancellation and send a confirmation.
